__tests__/lambda/lambdaResponses.test.ts

```diff
@@ -0,0 +1,103 @@
+import { lambdaResp, lambdaRespError } from '../../src/lambda'
```

Comment:
And about the tests made in another file?

Reply:
I'll move them to the new file as well